constants.h

```diff
@@ -0,0 +1,5 @@
+#define AVALUE (1)
+#define BVALUE (2)
```

Comment:
Please make these names more user friendly

Reply:
My naming conventions could use some improvement.

Reply:
I will update in the next pull request as we are already in QA test.